docs/pxc-strict-mode.md

```diff
@@ -187,13 +187,13 @@ At runtime, it is permitted to change `wsrep_replicate_myisam` to any value, but
 
 `ENFORCING` or `MASTER`
 
-At startup, if [`wsrep_replicate_myisam`](../wsrep-system-index.md#wsrep_replicate_myisam) is set to `ON`, an error is logged and startup is aborted.
+At startup, if [`wsrep_replicate_myisam`](wsrep-system-index.md#wsrep_replicate_myisam) is set to `ON`, an error is logged and startup is aborted.
 
-At runtime, any attempt to change [`wsrep_replicate_myisam`](../wsrep-system-index.md#wsrep_replicate_myisam) to `ON` fails and an error is logged.
+At runtime, any attempt to change [`wsrep_replicate_myisam`](wsrep-system-index.md#wsrep_replicate_myisam) to `ON` fails and an error is logged.
 
 !!! note
 
-    The [`wsrep_replicate_myisam`](../wsrep-system-index.md#wsrep_replicate_myisam) variable controls *replication* for MyISAM tables, and this validation only checks whether it is allowed. Undesirable operations for MyISAM tables are restricted using the Storage engine validation.
+    The [`wsrep_replicate_myisam`](wsrep-system-index.md#wsrep_replicate_myisam) variable controls *replication* for MyISAM tables, and this validation only checks whether it is allowed. Undesirable operations for MyISAM tables are restricted using the Storage engine validation.
 
 ### Binary log format
 
```

Comment:
We should not use acronyms in links, so it's better to remove pxc from the file name. pxc-strict-mode.md -> strict-mode.md

Reply:
Why not have acronyms in the links?

Reply:
Alex said it's bad for SEO

Reply:
I did a Google search on PXC. The PXC Strict Mode was the 3rd website returned.